### PR TITLE
Testing adding to install.rst 

### PR DESCRIPTION
**Pull request by mmariani123**
Hey Alex, 

I wanted to test the pull confirmation process I added a little markup to the beginning of [guide-chapter-install.rst] ,
I wanted to add a link to the minimal.ini and USAFEL-UchiTelle-small.pop , but I may not be linking correctly?

-Mike

### CODE REVIEW

File: website/docs/guide-chapter-usage.rst [hunk 1]
```diff
@@ -11,6 +11,11 @@ frequency data (as in :numref:`data-allelecount`),
 microsatellite data, SNP data, and nucleotide and amino acid sequence
 data.
 
+As mentioned in the installation chapter, a minimal working example 
+of a :ref:`configuration file (.ini) <https://github.com/mmariani123/pypop/blob/main/tests/data/USAFEL-UchiTelle-small.pop>`,
```
Comment (alexlancaster) [inline website/docs/guide-chapter-usage.rst:15]: You should change the base of the URL to be to the main repo, not your repo: https://github.com/alexlancaster/pypop/

Comment (alexlancaster) [reply website/docs/guide-chapter-usage.rst:15]: Also the `:ref:` syntax is for creating internal links to anchors within the document, for external URLs follow the syntax described here: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

**Review by alexlancaster — CHANGES_REQUESTED** [review]
Update link to main repo and follow this link syntax: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

**Comment by alexlancaster** [comment]
I just synched your branch with upstream `main` by doing a "rebase".

**Comment by alexlancaster** [comment]
The only reason this PR fails is because you don't have permission to update the beta docs site. I'll see if I can modify the GitHub action so it doesn't try that if the contributor does not have permission. The other option is that I add you as a contributor.

File: website/docs/guide-chapter-usage.rst (hunk 1 above)
Comment (alexlancaster) [inline website/docs/guide-chapter-usage.rst:15]:
```suggestion
of a`configuration file (.ini) <https://github.com/mmariani123/pypop/blob/main/tests/data/USAFEL-UchiTelle-small.pop>`_, 
```

File: website/docs/guide-chapter-usage.rst [hunk 2]
```diff
@@ -11,6 +11,11 @@ frequency data (as in :numref:`data-allelecount`),
 microsatellite data, SNP data, and nucleotide and amino acid sequence
 data.
 
+As mentioned in the installation chapter, a minimal working example 
+of a :ref:`configuration file (.ini) <https://github.com/mmariani123/pypop/blob/main/tests/data/USAFEL-UchiTelle-small.pop>`, 
+and a :ref:`population file (.pop) <https://github.com/mmariani123/pypop/blob/main/tests/data/minimal.ini>`,
```
Comment (alexlancaster) [inline website/docs/guide-chapter-usage.rst:16]:
```suggestion
and a `population file (.pop) <https://github.com/mmariani123/pypop/blob/main/tests/data/minimal.ini>`_, 
```

**Comment by alexlancaster** [comment]
@mmariani123  I have fixed the GitHub Action, so it now it skips trying to update the beta site if it's done from a fork. So now the checks "pass".  The only downside is you can't "preview" the website.  

I made some specific suggestions, you should be able to go through and add each suggestion in my review and batch commit them from the web interface - sort of like "accepting changes" in a Google doc or Word. That will update the branch. I can do a last check, and the merge into main.

File: website/docs/guide-chapter-usage.rst (hunk 1 above)
Comment (mmariani123) [reply website/docs/guide-chapter-usage.rst:15]: Thank you for the feedback Alex, I am attempting updates now!

Comment (alexlancaster) [reply website/docs/guide-chapter-usage.rst]: Cool. If you like the changes I suggested you can just commit those as-is and can then continue to make changes (if you need any more) on top of them.

File: website/docs/guide-chapter-usage.rst [hunk 3]
```diff
@@ -11,6 +11,11 @@ frequency data (as in :numref:`data-allelecount`),
 microsatellite data, SNP data, and nucleotide and amino acid sequence
 data.
 
+As mentioned in the installation chapter, a minimal working example 
+of a `configuration file (.ini) <https://github.com/alexlancaster/pypop/blob/main/tests/data/USAFEL-UchiTelle-small.pop>`,
```
Comment (alexlancaster) [inline website/docs/guide-chapter-usage.rst:15]:
```suggestion
of a `configuration file (.ini) <https://github.com/alexlancaster/pypop/blob/main/tests/data/USAFEL-UchiTelle-small.pop>`_, 
```

Close, but you need the underscore after the back tick.

File: website/docs/guide-chapter-usage.rst [hunk 4]
```diff
@@ -11,6 +11,11 @@ frequency data (as in :numref:`data-allelecount`),
 microsatellite data, SNP data, and nucleotide and amino acid sequence
 data.
 
+As mentioned in the installation chapter, a minimal working example 
+of a `configuration file (.ini) <https://github.com/alexlancaster/pypop/blob/main/tests/data/USAFEL-UchiTelle-small.pop>`, 
+and a `population file (.pop) <https://github.com/alexlancaster/pypop/blob/main/tests/data/minimal.ini>`,
```
Comment (alexlancaster) [inline website/docs/guide-chapter-usage.rst:16]:
```suggestion
and a `population file (.pop) <https://github.com/alexlancaster/pypop/blob/main/tests/data/minimal.ini>`_, 
```

**Review by alexlancaster — CHANGES_REQUESTED** [review]
Almost, see suggested changes. You still need the underscores (they work differently to the `:ref:` directives.

File: website/docs/guide-chapter-usage.rst (hunk 1 above)
Comment (alexlancaster) [reply website/docs/guide-chapter-usage.rst]: Ah, I see you made those changes. You're almost there - see my review, below.

**Review by alexlancaster — APPROVED** [review]
Looks good!

**Pull request merged**